README.md

```diff
@@ -1,10 +1,9 @@
 Dancer Sample App on OpenShift
 ============================
 
-This is a quickstart Dancer application for OpenShift v3.
+This is a quickstart Dancer application for OpenShift v3 that you can use as a starting point to develop your own application and deploy it on an [OpenShift](https://github.com/openshift/origin) cluster.
 
-The easiest way to install this application is to use the [OpenShift Instant Application](https://openshift.redhat.com/app/console/application_types).
-If you'd like to install it manually, follow [these directions](https://github.com/openshift/dancer-ex/blob/master/README#manual-installation).  
+If you'd like to install it, follow [these directions](https://github.com/openshift/rails-ex/blob/master/README.md#installation).  
 
 The steps in this assume that you have access to an OpenShift deployment; you must have an OpenShift deployment that you have access to in order to deploy this app.
```

Comment:
yeah I felt that way when i was doing this but i convinced myself it was saying it in sufficiently different ways as to leave it... but yeah, if you want to fix it up, i won't object.